CppCoreGuidelines.md

```diff
@@ -8720,7 +8720,7 @@ but at least we can see that something tricky is going on.
 ##### Note
 
 Unfortunately, `union`s are commonly used for type punning.
-We don't consider "sometimes, it works as expected" a strong argument.
```

Comment:
We're okay with improving this wording, but "conclusive" would be a better word, would you please use that.

Reply:
replaced with suggestion